<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyScannerClass</class>
 <widget class="QMainWindow" name="KeyScannerJiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KeyScanner</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="3,1">
      <item>
       <widget class="QGroupBox" name="gBoix_ImageDisplay">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2" stretch="9,1">
           <item>
            <widget class="QLabel" name="label_imgDisplay_1">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>999</width>
               <height>999</height>
              </size>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_showBtn" native="true"/>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <widget class="Line" name="line_rightDisplay">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="vLayout_SideBar" stretch="4,16,4,2">
        <item>
         <widget class="QGroupBox" name="groupBox_head">
          <property name="styleSheet">
           <string notr="true">QGroupBox {
    font-size: 20px;
    font-weight: bold;
    background-color: rgb(81, 81, 81);
    border: 1px solid #e0e0e0;
    border-radius: 2px;
	border-radius: 15px
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 5px;
    color: #2c3e50;
}</string>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3" stretch="2,0,3">
           <item>
            <layout class="QHBoxLayout" name="hLayout_title" stretch="3,2">
             <item>
              <widget class="QLabel" name="label_title">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">QLabel {
       font-size: 30px;        
       font-weight: bold;        
       color: rgb(255, 255, 255);         
       padding: 5px 5px;     
       border-bottom: 2px solid #cccccc; 
    }</string>
               </property>
               <property name="text">
                <string>钥匙检测</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pbtn_exit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
    color: white;
    border: none;
    border-radius: 5px;
    font-size: 14px;
    qproperty-icon: url(:/icons/closeBlack.png);  
    qproperty-iconSize: 50px 50px;
}

QPushButton:hover {
    background-color: #ff6666;
}

QPushButton:pressed {
    background-color: #cc0000;
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="shortcut">
                <string notr="true"/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="Line" name="line_2">
             <property name="styleSheet">
              <string notr="true">QLine {
       color: rgb(194, 194, 194);       
    }</string>
             </property>
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <layout class="QVBoxLayout" name="vLayout_cameraStates" stretch="3,2">
               <item>
                <widget class="QLabel" name="label_cameraStateTitle">
                 <property name="styleSheet">
                  <string notr="true">QLabel {
       font-size: 20px;        
       font-weight: bold;        
       color: rgb(232, 232, 232);         
       padding: 5px 5px;     
       border-bottom: 2px solid #cccccc; 
    }</string>
                 </property>
                 <property name="text">
                  <string>相机状态</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QGridLayout" name="gLayout_cameraStates">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_camera1State">
                   <property name="styleSheet">
                    <string notr="true">QLabel {
       font-size: 18px;        
       font-weight: bold;        
		padding: 5px 5px;  
    }</string>
                   </property>
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="vLayout_cardState" stretch="3,2">
               <item>
                <widget class="QLabel" name="label_cardStateTitle">
                 <property name="styleSheet">
                  <string notr="true">QLabel {
       font-size: 20px;        
       font-weight: bold;        
       color: rgb(232, 232, 232);         
       padding: 5px 5px;     
       border-bottom: 2px solid #cccccc; 
    }</string>
                 </property>
                 <property name="text">
                  <string>板卡状态</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_cardState">
                 <property name="styleSheet">
                  <string notr="true">QLabel {
       font-size: 18px;        
       font-weight: bold;        
padding: 5px 5px;  
    }</string>
                 </property>
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="gBox_infor">
          <property name="styleSheet">
           <string notr="true">QGroupBox {
    font-size: 20px;
    font-weight: bold;
    background-color: rgb(255, 255, 255);
    border: 1px solid #e0e0e0;
    border-radius: 2px;
	border-radius: 15px
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 5px;
    color: #2c3e50;
}</string>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout" stretch="1,2,2,2,1">
           <item>
            <layout class="QHBoxLayout" name="hLayout_info" stretch="1,1">
             <item>
              <widget class="QLabel" name="label_info">
               <property name="styleSheet">
                <string notr="true">QLabel {
       font-size: 24px;        
       font-weight: bold;        
       color:  rgb(141, 141, 141);       
       padding: 5px 5px;     
       border-bottom: 2px solid #cccccc; 
    }</string>
               </property>
               <property name="text">
                <string>统计信息</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pbtn_resetProduct">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
    padding: 6px 14px;
    border: 2px solid #CCC;
    border-radius: 4px;
    background-color: white;
    color: #444;
    font-size: 24px; 
}

QPushButton:hover {
    border-color: #999;
    background-color: #F5F5F5;
}

QPushButton:pressed {
    border-color: #777;
    background-color: #EEE;
}</string>
               </property>
               <property name="text">
                <string>产量清零</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="hLayout_produceTotal" stretch="1,2">
             <item>
              <widget class="QLabel" name="label_produceTotal">
               <property name="font">
                <font>
                 <pointsize>-1</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QLabel {
       font-size: 24px;        
       font-weight: bold;        
       color:  rgb(141, 141, 141);       
       padding: 5px 5px;     
       border-bottom: 2px solid #cccccc; 
    }</string>
               </property>
               <property name="text">
                <string>生产总量</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_produceTotalValue">
               <property name="styleSheet">
                <string notr="true">QLabel {
       font-size: 24px;        
       font-weight: bold;        
       color:  rgb(89, 89, 89);       
       padding: 5px 5px;     
       border-bottom: 2px solid #cccccc; 
    }</string>
               </property>
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="hLayout_wasteProducts" stretch="1,2">
             <item>
              <widget class="QLabel" name="label_wasteProducts">
               <property name="styleSheet">
                <string notr="true">QLabel {
       font-size: 24px;        
       font-weight: bold;        
       color:  rgb(141, 141, 141);       
       padding: 5px 5px;     
       border-bottom: 2px solid #cccccc; 
    }</string>
               </property>
               <property name="text">
                <string>废品总量</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_wasteProductsValue">
               <property name="styleSheet">
                <string notr="true">QLabel {
       font-size: 24px;        
       font-weight: bold;        
       color:   rgb(255, 0, 0);      
       padding: 5px 5px;     
       border-bottom: 2px solid #cccccc; 
    }</string>
               </property>
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QRadioButton" name="rbtn_debug">
               <property name="styleSheet">
                <string notr="true">QRadioButton {
    spacing: 8px;
    font-size: 20px;
    color: #333;
    font-weight: bold;        
    color:  rgb(0, 0, 0);       

}

/* 未选中状态 */
QRadioButton::indicator {
    width: 20px;
    height: 20px;
    border-radius: 10px;
    border: 2px solid #999;
    background: #ffffff;
}

/* 悬停效果 */
QRadioButton::indicator:hover {
    border: 2px solid #666;
}

/* 选中状态 */
QRadioButton::indicator:checked {
    border: 2px solid #2196F3;
}

/* 选中时的内圆 */
QRadioButton::indicator:checked {
    background-color: qradialgradient(
        cx:0.5, cy:0.5,
        radius:0.4,
        fx:0.5, fy:0.5,
        stop:0 #2196F3,
        stop:1 transparent
    );
}

/* 禁用状态 */
QRadioButton::indicator:disabled {
    border: 2px solid #ccc;
    background: #eee;
}

/* 点击动画效果 */
QRadioButton::indicator:pressed {
    border: 2px solid #1565C0;
    background-color: qradialgradient(
        cx:0.5, cy:0.5,
        radius:0.5,
        fx:0.5, fy:0.5,
        stop:0 #BBDEFB,
        stop:1 transparent
    );
}</string>
               </property>
               <property name="text">
                <string>调试模式</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="ckb_shibiekuang">
               <property name="styleSheet">
                <string notr="true">QCheckBox {
    spacing: 8px;
    font-size: 20px;
    color: #333;
    font-weight: bold;        
    color:  rgb(0, 0, 0);       
}

/* 未选中状态 */
QCheckBox::indicator {
    width: 20px;
    height: 20px;
    border-radius: 4px; /* 方形，圆角可适当调整 */
    border: 2px solid #999;
    background: #ffffff;
}

/* 悬停效果 */
QCheckBox::indicator:hover {
    border: 2px solid #666;
}

/* 选中状态 */
QCheckBox::indicator:checked {
    border: 2px solid #2196F3;
    background-color: qradialgradient(
        cx:0.5, cy:0.5,
        radius:0.4,
        fx:0.5, fy:0.5,
        stop:0 #2196F3,
        stop:1 transparent
    );
}

/* 禁用状态 */
QCheckBox::indicator:disabled {
    border: 2px solid #ccc;
    background: #eee;
}

/* 选中时的文字颜色 */
QCheckBox:checked {
    color: #2196F3;
    font-weight: 500;
}

/* 点击动画效果 */
QCheckBox::indicator:pressed {
    border: 2px solid #1565C0;
    background-color: qradialgradient(
        cx:0.5, cy:0.5,
        radius:0.5,
        fx:0.5, fy:0.5,
        stop:0 #BBDEFB,
        stop:1 transparent
    );
}</string>
               </property>
               <property name="text">
                <string>识别框</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="ckb_wenzi">
               <property name="styleSheet">
                <string notr="true">QCheckBox {
    spacing: 8px;
    font-size: 20px;
    color: #333;
    font-weight: bold;        
    color:  rgb(0, 0, 0);       
}

/* 未选中状态 */
QCheckBox::indicator {
    width: 20px;
    height: 20px;
    border-radius: 4px; /* 方形，圆角可适当调整 */
    border: 2px solid #999;
    background: #ffffff;
}

/* 悬停效果 */
QCheckBox::indicator:hover {
    border: 2px solid #666;
}

/* 选中状态 */
QCheckBox::indicator:checked {
    border: 2px solid #2196F3;
    background-color: qradialgradient(
        cx:0.5, cy:0.5,
        radius:0.4,
        fx:0.5, fy:0.5,
        stop:0 #2196F3,
        stop:1 transparent
    );
}

/* 禁用状态 */
QCheckBox::indicator:disabled {
    border: 2px solid #ccc;
    background: #eee;
}

/* 选中时的文字颜色 */
QCheckBox:checked {
    color: #2196F3;
    font-weight: 500;
}

/* 点击动画效果 */
QCheckBox::indicator:pressed {
    border: 2px solid #1565C0;
    background-color: qradialgradient(
        cx:0.5, cy:0.5,
        radius:0.5,
        fx:0.5, fy:0.5,
        stop:0 #BBDEFB,
        stop:1 transparent
    );
}</string>
               </property>
               <property name="text">
                <string>文字</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="hLayout_remove">
             <item>
              <widget class="QRadioButton" name="rbtn_removeFunc">
               <property name="styleSheet">
                <string notr="true">QRadioButton {
    spacing: 8px;
    font-size: 20px;
    color: #333;
    font-weight: bold;        
    color:  rgb(0, 0, 0);       

}

/* 未选中状态 */
QRadioButton::indicator {
    width: 20px;
    height: 20px;
    border-radius: 10px;
    border: 2px solid #999;
    background: #ffffff;
}

/* 悬停效果 */
QRadioButton::indicator:hover {
    border: 2px solid #666;
}

/* 选中状态 */
QRadioButton::indicator:checked {
    border: 2px solid #2196F3;
}

/* 选中时的内圆 */
QRadioButton::indicator:checked {
    background-color: qradialgradient(
        cx:0.5, cy:0.5,
        radius:0.4,
        fx:0.5, fy:0.5,
        stop:0 #2196F3,
        stop:1 transparent
    );
}

/* 禁用状态 */
QRadioButton::indicator:disabled {
    border: 2px solid #ccc;
    background: #eee;
}

/* 选中时的文字颜色 */
QRadioButton:checked {
    color: #2196F3;
    font-weight: 500;
}

/* 点击动画效果 */
QRadioButton::indicator:pressed {
    border: 2px solid #1565C0;
    background-color: qradialgradient(
        cx:0.5, cy:0.5,
        radius:0.5,
        fx:0.5, fy:0.5,
        stop:0 #BBDEFB,
        stop:1 transparent
    );
}</string>
               </property>
               <property name="text">
                <string>剔除功能</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="gBox_set">
          <property name="styleSheet">
           <string notr="true">QGroupBox {
    font-size: 20px;
    font-weight: bold;
    background-color: rgb(255, 255, 255);
    border: 1px solid #e0e0e0;
    border-radius: 2px;
	border-radius: 15px
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 5px;
    color: #2c3e50;
}</string>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="pbtn_start">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
    padding: 6px 14px;
    border: 2px solid #CCC;
    border-radius: 4px;
    background-color: white;
    color: #444;
    font-size: 24px; 
}

QPushButton:hover {
    border-color: #999;
    background-color: #F5F5F5;
}

QPushButton:pressed {
    border-color: #777;
    background-color: #EEE;
}</string>
             </property>
             <property name="text">
              <string>启动</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbtn_set">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
    padding: 6px 14px;
    border: 2px solid #CCC;
    border-radius: 4px;
    background-color: white;
    color: #444;
    font-size: 24px; 
}

QPushButton:hover {
    border-color: #999;
    background-color: #F5F5F5;
}

QPushButton:pressed {
    border-color: #777;
    background-color: #EEE;
}</string>
             </property>
             <property name="text">
              <string>设置</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="gBox_companyInfo">
          <property name="styleSheet">
           <string notr="true">QGroupBox {
    font-size: 20px;
    font-weight: bold;
    background-color: rgb(255, 255, 255);
    border: 1px solid #e0e0e0;
    border-radius: 2px;
	border-radius: 15px
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 5px;
    color: #2c3e50;
}</string>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_companyInfo">
             <property name="styleSheet">
              <string notr="true">QLabel {
       font-size: 18px;        
       font-weight: bold;        
       color:  rgb(141, 141, 141);       
       padding: 5px 5px;     
       border-bottom: 2px solid #cccccc; 
    }</string>
             </property>
             <property name="text">
              <string>暂无报警信息</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
